resources/views/components/progress-bar/index.blade.php

```diff
@@ -1,21 +1,29 @@
 @props(['checkout' => true])
-<div v-cloak class="flex items-center space-x-[18px] mb-[5px] sm:mb-0 text-sm">
-    @if($checkout)
-        <span class="antialiased font-medium whitespace-nowrap text-inactive">Stap @{{ checkout.step }} van de 4</span>
+<div
+    class="flex items-center space-x-[18px] text-13"
+    v-cloak
+>
+    @if ($checkout)
+        <span class="text-ct-inactive whitespace-nowrap font-medium">@lang('Step') @{{ checkout.step }} @lang('out of 4')</span>
     @else
-        <span class="antialiased font-medium whitespace-nowrap text-inactive">Stap 4 van de 4</span>
+        <span class="text-ct-inactive whitespace-nowrap font-medium">@lang('Step 4 out of 4')</span>
     @endif
-    @foreach(['Cart', 'Credentials', 'Payment', 'Confirmation'] as $stepTitle)
-        @if($checkout)
+    @foreach (['Cart', 'Credentials', 'Payment', 'Confirmation'] as $stepTitle)
+        @if ($checkout)
             <div
+                class="cursor-pointer"
                 v-on:click="goToStep({{ $stepTitle == 'Credentials' ? 2 : $loop->index }})"
-                class="hover:cursor-pointer"
-                :class="{'w-[12px] h-[12px] bg-accent': {{ $loop->index + 1 }} <= checkout.step, 'w-[12px] h-[12px] bg-border': {{ $loop->index + 1 }} > checkout.step, 'w-[20px] h-[20px] border-[4px] border-blue-110': {{ $loop->index + 1 }} === checkout.step}">
+                :class="{
+                    'w-[12px] h-[12px] bg-ct-accent rounded': {{ $loop->index + 1}} <= checkout.step,
+                    'w-[12px] h-[12px] bg-ct-border rounded': {{ $loop->index + 1 }} > checkout.step,
+                    'w-[20px] h-[20px] border-4 border-ct-accent-100 rounded-[6px]': {{ $loop->index + 1 }} === checkout.step
+                }"
+            >
             </div>
         @else
-        <div
-            v-on:click="goToStep({{ $stepTitle == 'Credentials' ? 2 : $loop->index }})"
-            class="w-[12px] h-[12px] bg-blue w-[20px] h-[20px] border-[4px] border-blue-110 hover:cursor-pointer"
+            <div
+                class="w-[20px] h-[20px] border-4 border-ct-accent-100 rounded-[6px] cursor-pointer"
+                v-on:click="goToStep({{ $stepTitle == 'Credentials' ? 2 : $loop->index }})"
```

Comment:
@royduin isn't the 2 in this case also hardcoded? if the steps change in the future or if we make the steps configurable?